### PR TITLE
Email posts 2

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: lib/community/solicitor.js [hunk 1]
```diff
@@ -46,13 +57,17 @@ var queueEmail = function (emailData) {
 }
 
 Solicitor.sendToUser = function (user, community) {
-  return queueEmail(emailData(user, community))
+  if (user.get('settings').email_solicitation_preference) {
+    return queueEmail(emailData(user, community))
+  } else {
+    return Promise.resolve()
+  }
```
Comment (levity) [inline lib/community/solicitor.js]: Another way to do this is to wrap the entire method in `Promise.method` and then just return nothing if the user doesn't have the setting turned on

Comment (robbiecarlton) [inline lib/community/solicitor.js]: Do you have a preference for the Promise.method way?
The above is easy for me to read

**Comment by robbiecarlton** [comment]
@levity good to merge?

File: lib/community/solicitor.js (hunk 1 above)
Comment (levity) [inline lib/community/solicitor.js]: It's fine, just FYI. Would be a bit more readable to use Promise.method in cases with more branching.

**Comment by levity** [comment]
:shipit:

**Pull request merged**